src/pages/login/LoginVscodeCallback.tsx

```diff
@@ -41,6 +78,30 @@ const LoginVscodeCallback: React.FC = () => {
         </div>
       </Card>
     </div>
+  ) : (
+    <div className={classNames(classes['Login'], Classes.DARK)}>
+      <Card elevation={Elevation.FOUR}>
+        <div>
+          <div className={classes['login-header']}>
+            <H4>
+              <Icon className={classes['login-icon']} icon={IconNames.LOG_IN} />
+              Sign in with SSO
```

Comment:
Do you know why this page is only shown when the user is not logged in? Otherwise, the frontend is redirected to `/` immediately. I can't figure this out